src/software/geom/util_test.cpp

```diff
@@ -253,145 +253,144 @@ TEST(GeomUtilTest, test_angle_sweep_circles)
     obs.push_back(Point(-9, 10));
     obs.push_back(Point(9, 10));
 
-    std::optional<std::pair<Point, Angle>> testpair_opt =
+    std::optional<Shot> testshot_opt =
         angleSweepCircles(Point(0, 0), Point(10, 10), Point(-10, 10), obs, 1.0);
 
     // We expect to get a result
-    ASSERT_TRUE(testpair_opt);
+    ASSERT_TRUE(testshot_opt);
 
-    std::pair<Point, Angle> testpair = *testpair_opt;
+    Shot testshot = *testshot_opt;
 
-    EXPECT_TRUE((testpair.first.norm() - Point(0, 1)).len() < 0.0001);
-    EXPECT_NEAR(75.449, testpair.second.toDegrees(), 1e-4);
+    EXPECT_TRUE((testshot.point.norm() - Point(0, 1)).len() < 0.0001);
+    EXPECT_NEAR(75.449, testshot.angle.toDegrees(), 1e-4);
```

Comment:
Similar here, should be `testshot.getPoint()` and `testshot.getAngle()`. Same for all the following places where you need access to `point` and `angle` from a `shot` object.

Reply:
Updated